UMLController.h

```diff
@@ -58,6 +58,10 @@ class UMLController
 
         void delete_class();
 
+        void rename_class();
+
+        void load_json();
```

Comment:
Would it also be possible to get saving as json in this PR too? The model has a function for that as well.